### PR TITLE
Improve handling of linking translations taking into account the stat…

### PR DESCRIPTION
**Pull request by sneridagh**
…e of the target. Restricting it completely for LRFs. Adding a transaction note to the action if it succeeds.

### CODE REVIEW

File: src/plone/restapi/services/multilingual/pam.py [hunk 1]
```diff
@@ -92,25 +93,44 @@ def reply(self):
...
         current_translation = manager.get_translation(target_language)
+        target_manager = ITranslationManager(target)
+        target_translation = target_manager.get_translation(target_language)
         if current_translation is not None:
             self.request.response.setStatus(400)
             return dict(
                 error=dict(
                     type="BadRequest",
-                    message="Already translated into language {}".format(
+                    message="Source already translated into language {}".format(
```
Comment (ericof) [inline src/plone/restapi/services/multilingual/pam.py:114]: use f-strings

File: src/plone/restapi/services/multilingual/pam.py [hunk 2]
```diff
@@ -92,25 +93,44 @@ def reply(self):
...
+                        target_language
+                    ),
+                )
+            )
+        if target_translation is not None:
+            self.request.response.setStatus(400)
+            return dict(
+                error=dict(
+                    type="BadRequest",
+                    message="Target already translated into language {}".format(
```
Comment (ericof) [inline src/plone/restapi/services/multilingual/pam.py:124]: use f-strings

File: src/plone/restapi/services/multilingual/pam.py [hunk 3]
```diff
@@ -92,25 +93,44 @@ def reply(self):
             )
 
         target = self.get_object(id_)
+        import pdb;pdb.set_trace()
```
Comment (ericof) [inline src/plone/restapi/services/multilingual/pam.py:96]: This should not be here :-)

**Review by reebalazs — APPROVED** [review]

**Comment by sneridagh** [comment]
@ericof proper PR now :)

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Review by tisto — APPROVED** [review]

**Pull request merged**